### ISSUE TITLE
Two sample t-test

### ISSUE THREAD

**Issue by arnodelorme** [LIMO-EEG-Toolbox/limo_tools#114]
Does it make any sense to allow users to select a different LIMO 1-st level analysis to compare Betas. They should only compare Betas from the same 1-st level analysis. I can fix this. Just let me know.

Also, corrected a bug line 519 of limo_random_select

```
                sub_LIMO = load(cell2mat(fullfile(LIMO.data.data_dir{gp}{1}(sub),'LIMO.mat')));
```

instead of 

```
                sub_LIMO = load(cell2mat(fullfile(LIMO.data.data_dir{gp}(sub),'LIMO.mat')));
```

It is fine. I will do a pull request later.

**Comment by arnodelorme** [comment]
It might be that you can select beta and cons files, right?

**Comment by CPernet** [comment]
well, by default of course the beta or con are associated with a given LIMO.mat ; there are occasional cases for which you may need to select another one. In a multisession experiment, each session is automatically built and analyzed and we also (when design are identical) create between sessions contrasts - those need a LIMO file, so yes it makes sense (also some other edge cases I did for someone before)

**Comment by arnodelorme** [comment]
Yes, that makes sense for session contrast. Are there a set of beta for session 1 and another set for session 2?

**Comment by CPernet** [comment]
Well yes each session has betas, and one folder up you have between sessions contrasts
It turns out I create a LIMO.mat for that too! makes sense because the channel array might be a little different. So while the code calling the 'LIMO-mat' should not change, users do not have to pick it up (just get the one next to file - and it not there prompt?)

https://github.com/sccn/eeglab/blob/develop/functions/studyfunc/std_limo.m#L575
https://github.com/LIMO-EEG-Toolbox/limo_tools/blob/master/limo_contrast_sessions.m#L157
